data/migrations/20200112151338_users-schema-update.js

```diff
@@ -0,0 +1,9 @@
+exports.up = function(knex) {
+  return knex.schema.table('users', table => {
+    table.timestamp('createdon').defaultTo(knex.fn.now());
+  });
+};
+
+exports.down = function(knex) {
```

Comment:
Refer to link https://devhints.io/knex on how to alter a table and drop a single column too